rskj-core/src/main/resources/config/regtest.conf

```diff
@@ -24,12 +24,14 @@ peer {
         # the peer window will show
         # only what retrieved by active
         # peer [true/false]
-        enabled = false
+        enabled = true
```

Comment:
is it by accident? we don't need this, do we?

Reply:
Yes, I forgot to remove it. Fixed, thx!